File: scripts/update_lockfile.py
import subprocess
import sys

try:
    print("Running pnpm install --no-frozen-lockfile...")
    result = subprocess.run(
        ["pnpm", "install", "--no-frozen-lockfile"],
        cwd="/vercel/share/v0-project",
        capture_output=True,
        text=True
    )
    
    print(result.stdout)
    if result.stderr:
        print("STDERR:", result.stderr)
    
    if result.returncode == 0:
        print("✓ Lockfile updated successfully!")
    else:
        print(f"✗ Error: pnpm install failed with return code {result.returncode}")
        sys.exit(1)
except Exception as e:
    print(f"✗ Error: {e}")
    sys.exit(1)

File: scripts/update-lockfile.mjs
import { execSync } from 'child_process';

try {
  console.log('Updating pnpm-lock.yaml...');
  execSync('pnpm install --no-frozen-lockfile', {
    stdio: 'inherit',
    cwd: '/vercel/share/v0-project'
  });
  console.log('Lockfile updated successfully!');
} catch (error) {
  console.error('Error updating lockfile:', error.message);
  process.exit(1);
}
